tests/end_to_end/test_emotet/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,99 @@
+- name: Test agent configuration
+  hosts: wazuh-windows
+  tasks:
+
+    - name: Create temp folder
+      win_file:
+        path: C:\temp
+        state: directory
+
+    - name: Copy ossec.conf
+      ansible.windows.win_copy:
+        src: C:\Program Files (x86)\ossec-agent\ossec.conf
+        dest: C:\temp
+        remote_src: true
+
+    - name: Install sysmon
```

Comment:
```suggestion
    - name: Download sysmon
```

Reply:
Done in ed8a4619cf33626e52fad3a22a38531f8e3a35c1